.ci.yaml

```diff
@@ -4618,22 +4618,12 @@ targets:
 
   - name: Mac_ios platform_views_scroll_perf_ios__timeline_summary
     recipe: devicelab/devicelab_drone
-    presubmit: false
```

Comment:
this is just a trick for CI to run the test. I will create a new task `platform_views_scroll_perf_ad_banners__timeline_summary` rather than reusing existing task